tests/test_data/test_autoaug.py

```diff
@@ -0,0 +1,326 @@
+import copy
+import os.path as osp
+
+import numpy as np
+import pytest
+from mmcv.utils import build_from_cfg
+
+from mmdet.datasets.builder import DATASETS, PIPELINES
+
+
+def test_translate():
+    # test assertion for invalid value of level
+    with pytest.raises(AssertionError):
+        transform = dict(type='Translate', level=-1)
+        build_from_cfg(transform, PIPELINES)
+
+    # test assertion for invalid type of level
+    with pytest.raises(AssertionError):
+        transform = dict(type='Translate', level=[1])
+        build_from_cfg(transform, PIPELINES)
+
+    # test assertion for invalid prob
+    with pytest.raises(AssertionError):
+        transform = dict(type='Translate', level=1, prob=-0.5)
+        build_from_cfg(transform, PIPELINES)
+
+    # test assertion for the num of elements in tuple fill_val
+    with pytest.raises(AssertionError):
+        transform = dict(
+            type='Translate', level=1, fill_val=(128, 128, 128, 128))
+        build_from_cfg(transform, PIPELINES)
+
+    # test ValueError for invalid type of fill_val
+    with pytest.raises(ValueError):
+        transform = dict(type='Translate', level=1, fill_val=[128, 128, 128])
+        build_from_cfg(transform, PIPELINES)
+
+    # test assertion for invalid value of fill_val
+    with pytest.raises(AssertionError):
+        transform = dict(type='Translate', level=1, fill_val=(128, -1, 256))
+        build_from_cfg(transform, PIPELINES)
+
+    # test assertion for invalid value of axis
+    with pytest.raises(AssertionError):
+        transform = dict(type='Translate', level=1, fill_val=128, axis='z')
+        build_from_cfg(transform, PIPELINES)
+
+    # test assertion for invalid type of max_translate_offset
+    with pytest.raises(AssertionError):
+        transform = dict(
+            type='Translate',
+            level=1,
+            fill_val=128,
+            max_translate_offset=(250., ))
+        build_from_cfg(transform, PIPELINES)
+
+    # construct CocoDataset as the dataset example for testing
+    train_pipeline = [
+        dict(type='LoadImageFromFile'),
+        dict(
+            type='LoadAnnotations',
+            with_bbox=True,
+            with_mask=True,
+            with_seg=True),
+    ]
+    # TODO handle data_root, assume folder 'coco' is in path test/data
+    data_root = osp.join(osp.dirname(__file__), '../data/coco/')
+    val_data = dict(
+        type='CocoDataset',
+        ann_file=osp.join(data_root, 'annotations/instances_val2017.json'),
+        img_prefix=osp.join(data_root, 'val2017/'),
+        seg_prefix=osp.join(data_root, 'stuffthingmaps/val2017/'),
+        pipeline=train_pipeline,
+        test_mode=False)
```

Comment:
It may be better to upload some dummy data and put it into 'tests/data'. Otherwise, it can't pass CI tests.

Reply:
Hi, I will upload some dummy data and put it into 'tests/data' folder.

Reply:
Some dummy COCO data (two images and its corresponding annotations) is uploaded under folder 'tests/data/coco_dummy', for the purpose of testing image, bboxes, masks and segmentations.